### PR TITLE
QR Code PIX

### PR DESCRIPTION
**Pull request by RcleydsonR**
Implementation of PIX QR Code generator.
Also implemented a component that allow the user to generate a pix qr code in UI.

### CODE REVIEW

File: src/views/QrCodeForm.vue [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+            <span v-if="errors['pixRequiredError']" class="required-error"
+              >(Esse campo é obrigatório)</span
+            >
+          </div>
+          <input
+            type="text"
+            name="pixKey"
+            id="pixKey"
+            class="form-input"
+            v-model="PixModel['pixKey']"
```
[the remark this answers is not in the document]
Comment (RcleydsonR) [reply src/views/QrCodeForm.vue:75]: I'll see if ref has a special behavior, if not, I apply these changes.

File: src/views/QrCodeForm.vue [hunk 2]
```diff
@@ -0,0 +1,217 @@
+<script setup lang="ts">
+import { Pix } from "../utils/QrCodePix";
+import { ref } from "vue";
+
+const PixModel = ref({
```
[the remark this answers is not in the document]
Comment (RcleydsonR) [reply src/views/QrCodeForm.vue:5]: I think it's better, it was just a mistake, thanks to seeing this.

File: src/views/QrCodeForm.vue [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      </div>
+    </div>
+    <div
+      v-if="toggleModal"
+      class="fixed z-40 inset-0 opacity-25 bg-black"
+    ></div>
+  </div>
+</template>
+
+<style>
```
[the remark this answers is not in the document]
Comment (RcleydsonR) [reply src/views/QrCodeForm.vue]: Fixed

File: src/views/QrCodeForm.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (RcleydsonR) [reply src/views/QrCodeForm.vue]: Fixed

File: src/utils/QrCodePix.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
+import qrcode from "qrcode";
```
[the remark this answers is not in the document]
Comment (RcleydsonR) [reply src/utils/QrCodePix.ts:1]: Fixed

File: src/views/QrCodeForm.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (RcleydsonR) [reply src/views/QrCodeForm.vue]: Fixed

**Review by Ronyell — APPROVED** [review]

**Pull request merged**